create_gz_vendor_pkg/templates/CMakeLists.txt.jinja

```diff
@@ -16,6 +16,17 @@ set(LIB_NAME_COMP_PREFIX ${LIB_NAME})
 set(LIB_NAME_FULL ${LIB_NAME}${LIB_VER_MAJOR})
 set(LIB_VER ${LIB_VER_MAJOR}.${LIB_VER_MINOR}.${LIB_VER_PATCH})
 
+option(VENDOR_FROM_LIB_VCS_REF
```

Comment:
We are using environment variables below for configuring how strict the version match should be. Any reason for using a CMake option here?

Reply:
Fair enough. Thanks for explaining.